src/Compilers/CSharp/Portable/Binder/LookupOptions.cs

```diff
@@ -110,6 +110,11 @@ internal enum LookupOptions
         /// Do not consider symbols that are parameters.
         /// </summary>
         MustNotBeParameter = 1 << 16,
+
+        /// <summary>
+        /// Broaden the lookup to include members from extension types.
+        /// </summary>
+        SearchInExtensionTypes = 1 << 17,
```

Comment:
I might be missing something, but it feels to me, that the rule will be to search in extension types, and ignoring extension types will be an exception. Perhaps we should have an "ignore" flag instead?

Reply:
The reason I went with an "include" flag instead of an "ignore" flag is because we used an "include" flag for extension methods. But I agree that extensions types may be sufficiently different from extension methods (more common), so maybe inverting makes sense.
I'll add a PROTOTYPE